db/ddl.sql

```diff
@@ -117,6 +117,7 @@ CREATE TABLE orbital_responses (
   params_terminal_id varchar(255) DEFAULT NULL,
   params_tx_ref_idx varchar(255) DEFAULT NULL,
   params_tx_ref_num varchar(255) DEFAULT NULL,
+  params_trace_number varchar(16) DEFAULT NULL,
```

Comment:
Is that always guaranteed to be less than 16 chars? If not, might as well set it to 255 (storage is cheap).